components/atoms/Label/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { styled } from 'nativewind';
+import { Text as NativeText } from 'react-native';
+import classNames from 'classnames';
+import { View } from 'components/atoms/View';
+
+const StyledText = styled(NativeText);
+
+interface IfLabelProps {
+    text?: string;
+    textSize?: 'xs' | 'sm' | 'base' | 'lg' | 'xl' | '2xl' | '3xl';
```

Comment:
Komponent labelki oprzyj o komponent Textu, który posiada zdefiniowane wielkości tekstu.
Druga sprawa, że labelki mają mieć chyba 2 font size'y, zajrzyj na design i zdefiniuj już na gotowo ich wielkość. 
Nie myl komponentu Textu z Label, Label to już gotowy wygląd.

Reply:
aha przyjałem